File: src/SOP8/광양/EgovLogin/base-account-ibll/src/main/java/egovframework/base/account/ibll/request/RequestLogin.java
package egovframework.base.account.ibll.request;

public class RequestLogin {

    private String value = "";
    private String key = "";
    private String externalLoginUrl = null;
    private boolean autoLogin = false;

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getExternalLoginUrl() { return externalLoginUrl; }

    public void setExternalLoginUrl(String externalLoginUrl) { this.externalLoginUrl = externalLoginUrl; }

    public boolean getAutoLogin() { return autoLogin; }

    public void setAutoLogin(boolean autoLogin) { this.autoLogin = autoLogin; }
}

File: src/SOP8/Base/Account/Base.Account.IBLL/Request/RequestLogin.cs
namespace Base.Account.IBLL.Request
{
    public class RequestLogin
    {
        private string m_strValue = "";
        private string m_strKey = "";

        public string Value
        {
            get { return m_strValue; }
            set { m_strValue = value; }
        }

        public string Key
        {
            get { return m_strKey; }
            set { m_strKey = value; }
        }
    }
}
